source/core/VirtualInstrument.cpp

```diff
@@ -564,7 +564,10 @@ void ClumpParseState::ResolveActualArgument(SubString* argument, void** ppData,
     // See if it is in the VI's locals or paramblock
     _actualArgumentType = _vi->GetVIElementAddressFromPath(argument, _vi, (void**)ppData, false);
     if (_actualArgumentType) {
-        _argumentState = kArgumentResolvedToVIElement;
+        if (_actualArgumentType->ElementUsageType() == kUsageTypeInput && _formalParameterType->ElementUsageType() != kUsageTypeInput)
```

Comment:
This is a good idea. I am not sure about kUsageTypeAlias situations but not likely used much in ViaTests and not likely used from DFIR at all.

Reply:
I don't think kUsageTypeAlias can show up in this context because _actualArgumentType has to be an input for the failure case, and inputs only show up in the parameter block.  Also, the SharedArray test does some testing of aliases and still works

Reply:
Got it. thanks.